_includes/head.html

```diff
@@ -16,6 +16,6 @@
 </script>
 
 <!-- For all browsers -->
-<link rel="stylesheet" href="{{ base_path }}/assets/css/main.css">
+<link rel="stylesheet" href="/assets/css/main.css">
```

Comment:
Why this base_path is removed? This will prevent it from being hosted in subdir.